### ISSUE TITLE
kRPCConnect error

### ISSUE THREAD

**Issue by gstavrinos** [BenChung/KRPC.jl#1]
Hey, thanks for the package! I started experimenting with it, but unfortunately, I hit an error right into the first command of my program!
When running `kRPCConnect("test")` I get the following error:

```
ERROR: MethodError: no method matching read(::TCPSocket, ::Type{UInt8}, ::Int64)
Closest candidates are:
  read(::Base.LibuvStream, ::Type{UInt8}) at stream.jl:751
  read(::IO, ::Type{UInt8}) at io.jl:182
  read(::AbstractString, ::Any...) at io.jl:295
  ...
Stacktrace:
 [1] RecvRawProto(::TCPSocket) at /home/gstavrinos/.julia/packages/kRPC/N4650/src/raw.jl:26
 [2] connect_or_error(::TCPSocket, ::kRPC.krpc.schema.ConnectionRequest) at /home/gstavrinos/.julia/packages/kRPC/N4650/src/connection.jl:11
 [3] kRPCConnect(::String, ::String, ::Int64, ::Int64) at /home/gstavrinos/.julia/packages/kRPC/N4650/src/connection.jl:47
 [4] kRPCConnect(::String) at /home/gstavrinos/.julia/packages/kRPC/N4650/src/connection.jl:40
 [5] top-level scope at none:0
```

Any ideas?

Thanks,

George

**Comment by BenChung** [comment]
Ah, this is due to a newer version of Julia than the one I originally was working on. I'll get a fix in soon, thanks!

I should warn you that this package isn't really in good shape for external use; as long as you can get through the setup process it should be fine, but please let me know if you run into problems.

**Comment by BenChung** [comment]
Should be fixed in 031d88d.

**Comment by gstavrinos** [comment]
Thanks for the quick fix! I will check it out later today and report back if I face any other problem! Cheers!
